docs/node/server-plugins.md

```diff
@@ -75,8 +81,12 @@ The plugin accepts all common options, and the following additional ones:
   context values are passed to the service implementation. See
   [Context values](./interceptors.md#context-values) for more information.
 
-Note that `@connectrpc/connect` and `@connectrpc/connect-node` are peer 
-dependencies of  `@connectrpc/connect-fastify`.
+:::note
+Over HTTP/2, Fastify can serve the Connect, gRPC, and gRPC-Web protocols with
+all of RPCs.
```

Comment:
```suggestion
all types of RPCs.
```

Reply:
Thanks for the catch! Updated to "all types of" in https://github.com/connectrpc/connectrpc.com/pull/174/commits/59858f1122828be774e94bbb58af767d6cd31c8d and https://github.com/connectrpc/connectrpc.com/pull/174/commits/655d5426b598b6f705af14b786da3d5ed7aa176a.